webapp/packages/plugin-connections/src/ConnectionForm/Options/ProviderPropertiesExtendedService.ts

```diff
@@ -0,0 +1,24 @@
+/*
+ * CloudBeaver - Cloud Database Manager
+ * Copyright (C) 2020-2025 DBeaver Corp and others
+ *
+ * Licensed under the Apache License, Version 2.0.
+ * you may not use this file except in compliance with the License.
+ */
+import { PlaceholderContainer } from '@cloudbeaver/core-blocks';
+import { injectable } from '@cloudbeaver/core-di';
+import type { IFormState } from '@cloudbeaver/core-ui';
+import type { IConnectionFormState } from '../IConnectionFormState.js';
+
+export type ProviderPropertiesExtendedFormProps = {
+  formState: IFormState<IConnectionFormState>;
+};
+
+@injectable(() => [])
+export class ProviderPropertiesExtendedService {
+  readonly container: PlaceholderContainer<ProviderPropertiesExtendedFormProps>;
```

Comment:
Please move it to the ConnectionFormService or OptionsPartService. There is no need for such decomposition